Runtime/Game/Requests/NotificationRequests.cs

```diff
@@ -28,6 +28,85 @@ public enum LootLockerNotificationContentKind
     };
 }
 
+namespace LootLocker.LootLockerStaticStrings
+{
+    /// <summary>
+    /// Possible types of notifications
+    /// </summary>
+    public struct LootLockerNotificationTypes
+    {
+        public static readonly string PullRewardAcquired = "pull.reward.acquired";
+    }
+
+    /// <summary>
+    /// Possible sources for notifications
+    /// </summary>
+    public struct LootLockerNotificationSources
+    {
+        public static readonly string Triggers = "triggers";
+        public struct Purchasing
+        {
+            public static readonly string SteamStore = "purchasing.steam_store";
+            public static readonly string AppleAppStore = "purchasing.apple_app_store";
+            public static readonly string GooglePlayStore = "purchasing.google_play_store";
+            public static readonly string LootLocker = "purchasing.lootlocker";
+        }
+    }
+
+    /// <summary>
+    /// The standard context keys to expect for different notification sources
+    /// </summary>
+    public struct LootLockerStandardContextKeys
+    {
+        /// <summary>
+        /// Standard context keys to expect when source is triggers
+        /// </summary>
+        public struct Triggers
+        {
+            public static readonly string TriggerID = "trigger_id";
+            public static readonly string TriggerKey = "trigger_key";
+            public static readonly string TriggerLimit = "trigger_limit";
```

Comment:
Maybe rename these to just id, key and limit? It is specified in the struct above that it is Triggers.

Reply:
SMORT

Reply:
Fixed